Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ubuntu:14.04
+MAINTAINER Oscar Fanelli <oscar.nesis@gmail.com>
```

Comment:
Why not? I saw that this is a common practice for those who create a Dockerfile. If it's a problem, please tell me.